src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/DatePicker/DatePickerSample.xaml

```diff
@@ -14,13 +14,19 @@
 	<controls:SampleControl SampleDescription="Sample for Date Picker">
 		<controls:SampleControl.SampleContent>
 			<DataTemplate>
-				<Grid Background="White">
+				<StackPanel>
 					<DatePicker Grid.Row="1"
 								Margin="15,10"
 								HorizontalAlignment="Center"
 								VerticalAlignment="Top">
 					</DatePicker>
-				</Grid>
+
+					<Button Content="Open DatePickerFlyout">
```

Comment:
This should go in a separate sample.

Reply:
I left it there because you can't test the placement with the current template of `DatePicker` , instead of having a <kbd>[2019 / 07 / 18]</kbd> that opens the flyout on click, you get the flyout (droid: calendar, ios: wheel picker) directly.
However, fixing the `DatePicker` template is not in the scope of this PR.

Reply:
Could you add a comment to that effect in the xaml? You can link to this issue: https://github.com/unoplatform/uno/issues/188